### DISCUSSION TITLE
How to select region in flameshot 12

### DISCUSSION THREAD

**Discussion by agutierrm** [flameshot-org/flameshot#2993] [General]
Hi there, I run flameshot 12 in laptop and 11 in my home computer. I have seen that in flameshot 11 I can select region with contextual menu in gnome but in 12, when I select take screenshot I cant select a rectangular region as I do in 11. This is normal? Thanks!

**Comment by mmahmoudian** [comment]
I guess one of your laptops (the one that works) have Gnome < 42 and the other one has Gnome >= 42 and Wayland. Please read this page and if it did not clear the issue for you, comment here with more details (OS, Gnome version, how you have installed Flameshot, How are you running Flameshot, what happens when you run `flameshot gui`)

**Comment by agutierrm** [comment]
Hi there, thanks for asking, where is running is Ubuntu 22.04, but with x11, not wayland. This is because Im running on a nvidia card with propietary driver.
XDG_SESSION_TYPE=x11

So, in my PC (the one running x11) when I press on the dock, and I select (Take screenshot) [Exec=flameshot gui --delay 500] the desktop goes grey and I can select with the mouse a rectangle. After this I can edit. This doesnt happens on my laptop :-(

**Reply by mmahmoudian** [reply]
Do you have the same X11 environment on your laptop? Do you have the same EXEC in the .desktop file? have you confirmed that you have Flameshot v12.1.0 on both machines?
